### ISSUE TITLE
Update examples to use simplified accounts.

### ISSUE THREAD

**Issue by Shaptic** [stellar/new-docs#360]
In stellar/go#3393, we simplified the account interface to be entirely string-based. The examples need to be updated accordingly.

For instance, in the examples for [sponsored reserves](https://developers.stellar.org/docs/glossary/sponsored-reserves/#examples), we **have** the following:

```go
txnbuild.BeginSponsoringFutureReserves{
	SourceAccount: &s1Account,
	SponsoredID:   addressA,
}
```

but we **need** the following:

```go
txnbuild.BeginSponsoringFutureReserves{
	SourceAccount: s1Account.AccountID,
	SponsoredID:   addressA,
}
```

**Comment by Godspowerogun** [comment]
Blockchain

**Comment by Shaptic** [comment]
This was closed in #411.
